wardrobe/timestamp/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+description = "A Tuxedo piece that allows block authors to include a wall-clock time stamp in a block via an inherent extrinsic."
+edition = "2021"
+name = "timestamp"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = { workspace = true }
+parity-scale-codec = { features = [ "derive" ], workspace = true }
+scale-info = { features = [ "derive" ], workspace = true }
+serde = { features = [ "derive" ], optional = true, workspace = true }
+sp-api = { default_features = false, workspace = true }
```

Comment:
Just as an FYI if in the Workspace we automatically add 'default-features = false' then we dont also need to do it here. We instead can just do 'sp-api = { workspace = true }'

Reply:
Nice! I made an issue for it https://github.com/Off-Narrative-Labs/Tuxedo/issues/122